.stylelintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "stylelint-config-sass-guidelines",
+  "plugins": [
+     "stylelint-suitcss",
+     "stylelint-scss"
+  ],
+  "rules": {
+    "max-nesting-depth": 3,
+    "scss/at-mixin-pattern": "^(terraConsumer-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$",
+    "suitcss/custom-property-no-outside-root": true,
+    "custom-property-pattern": [
+      "^(terra|terraConsumer)-(?:[A-Z][a-zA-Z0-9]+)*(?:-[a-z0-9][a-zA-Z0-9]+)*?(?:--[a-z0-9][a-zA-Z0-9-]+)*?$",
```

Comment:
You'll likely want to update this to match what we have in core: https://github.com/cerner/terra-core/blob/master/.stylelintrc#L11